packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanBar.css

```
@@ -44,6 +44,13 @@ limitations under the License.
   z-index: 1;
 }
 
+.SpanBar--criticalPath {
+  position: absolute;
+  top: 46%;
+  height: 10%;
+  z-index: 1;
```

If we want to overlay critical path on top of spans, let's increase the z-index to more than 1.
```suggestion
  z-index: 2;
```